src/main/java/uk/gov/hmcts/reform/ccd/document/am/model/UploadDocumentsCommand.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.hmcts.reform.ccd.document.am.model;
+
+import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.web.multipart.MultipartFile;
+import uk.gov.hmcts.reform.ccd.document.am.model.enums.Classifications;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+
+
+@Data
+public class UploadDocumentsCommand {
+
+    @NotNull(message = "Provide some files to be uploaded.")
+    @Size(min = 1, message = "Please provide at least one file to be uploaded.")
+    private List<MultipartFile> files;
+
+    @NotNull(message = "Please provide classification")
+    private Classifications classification;
```

Comment:
name `Classifications` should be singular. Otherwise this line reads misleadingly.